framework/src/executioners/Transient.C

```diff
@@ -203,6 +217,18 @@ Transient::Transient(const InputParameters & parameters)
     if (_num_steps == 0) // Always do one step in the first half
       _num_steps = 1;
   }
+
+  // Set up relaxation
+  if (_relax_factor != 1.0)
+  {
+    if (_relax_factor >= 2.0 || _relax_factor <= 0.0)
+      mooseError("The Picard iteration relaxation factor should be between 0.0 and 2.0");
+    NonlinearSystem & _nl_system = _fe_problem.getNonlinearSystem();
+    System & libmesh_nl_system = _nl_system.system();
+
+    // Store a copy of the previous solution here
+    libmesh_nl_system.add_vector("relax_previous", false, PARALLEL);
```

Comment:
You could also do: `_nl_system.addVector("relax_previous", false, PARALLEL);` and then you do not have to go via libMesh interface.